frontend/src/metabase/dashboard/components/DashCard/DashCard.tsx

```diff
@@ -164,7 +164,7 @@ function DashCard({
   );
 
   const isAction = isActionCard(mainCard);
-  const isEmbed = Utils.isJWT(dashcard.dashboard_id);
```

Comment:
Referring to my comment on `utils.ts` - I would much prefer to find out that `dashcard.dashboard_id` might have a problem with its type here, which would make debugging easier if `dashboard_id` somehow is messed up